tests/ceph_volume/test_ceph_volume_get_inventory_list.py

```diff
@@ -0,0 +1,52 @@
+from cli.exceptions import OperationFailedError
+from cli.utilities.containers import Container
+
+
+def get_inventory_list(node, output_format):
+    # Note: This cmd uses rhceph-5-rhel8:latest, the requirement is just to ensure whether
+    # the ceph inventory is able to list the details and any container image is sufficient
+    cmd = (
+        "--privileged --net=host --ipc=host --ulimit nofile=1024:4096 -v /run/lock/lvm:/run/lock/lvm:z -v "
```

Comment:
@ pranavprakash20 we can pass volume and image to run the method rather than passing it as cmd. 
@vamahaja what do you suggest?

Reply:
Done @adityaramteke